### ISSUE TITLE
rebar user/password hard-coded into the api container

### ISSUE THREAD

**Issue by rstarmer** [rackn/digitalrebar-deploy#450]
It appears that the rebar user and it's associated password are hard-coded into the rebar api container.  Can we get this resovled please? I will need this to take the system into production.

**Comment by galthaus** [comment]
We need to do this, but you can change it after the installation.

**Comment by galthaus** [comment]
Actually, what password do you mean?  The digitalrebar rebar user?

**Comment by rstarmer** [comment]
yeah rebar user rebar1 password,

example, in 23-wait-for-api-to-start.sh

while ! rebar -U rebar -P rebar1 ping; do

If I change the rebar password, the service will not restart.

and a couple of times in 24-build-initial-machine-keys.sh

R

Robert

**Comment by galthaus** [comment]
Hmm - yes - that is bad.  We need to convert those to internal auth calls.

**Comment by galthaus** [comment]
Didn't mean to close.

**Comment by rstarmer** [comment]
Yeah, I'm ok with this during the prototype phase, but this fails our
production security checks.


Robert
